packages/blade/src/components/Button/BaseButton/StyledBaseButton.native.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components/native';
+import getBaseButtonStyles from './getBaseButtonStyles';
+import type { StyledBaseButtonProps } from './StyledBaseButton';
+
+const StyledPressable = styled.Pressable(
+  ({ color, hoverColor, activeColor }: Omit<StyledBaseButtonProps, 'children' | 'onClick'>) =>
+    getBaseButtonStyles({ color, hoverColor, activeColor }),
+);
+
+const StyledBaseButton = ({
+  onClick,
+  color,
+  hoverColor,
+  activeColor,
+  children,
+}: StyledBaseButtonProps): React.ReactElement => {
+  return (
+    <StyledPressable
+      onPress={onClick}
+      color={color}
+      hoverColor={hoverColor}
+      activeColor={activeColor}
+      style={({ pressed }): { backgroundColor: string; borderColor: string } => ({
+        backgroundColor: pressed ? activeColor : color,
+        borderColor: pressed ? activeColor : color,
+      })}
```

Comment:
can't this be part of `getBaseButtonStyles`? or this is something explicitly that the `StyledPressable` needs as attributes?

Reply:
Will handle this better in later PRs

Reply:
any updates here?